tests/src/system/basic/WskBasicTests.scala

```diff
@@ -90,9 +90,38 @@ class WskBasicTests
         stdout should include regex ("""(?i)whisk API version\s+v1""")
     }
 
+    it should "validate default property values" in {
+        val tmpwskprops = File.createTempFile("wskprops", ".tmp")
+        val env = Map("WSK_CONFIG_FILE" -> tmpwskprops.getAbsolutePath())
+        try {
+            wsk.cli(Seq("property", "unset", "--auth", "--apihost", "--apiversion", "--namespace"), env = env)
+            var stdout = wsk.cli(Seq("property", "get", "--auth"), env = env).stdout
+            stdout should include regex ("""(?i)whisk auth\s*$""")      // default = empty string
+            stdout = wsk.cli(Seq("property", "get", "--apihost"), env = env).stdout
+            stdout should include regex ("""(?i)whisk API host\s*$""")  // default = empty string
+            stdout = wsk.cli(Seq("property", "get", "--namespace"), env = env).stdout
+            stdout should include regex ("""(?i)whisk namespace\s*_$""")// default = _
+        }
+        finally {
+            tmpwskprops.delete()
+        }
+    }
+
     it should "show api build version" in {
-        val stdout = wsk.cli(wskprops.overrides ++ Seq("property", "get", "--apibuild")).stdout
-        stdout should include regex ("""(?i)whisk API build\s+201.*""")
+        //val wskpropsBackup = TestUtils.backupWskProps()
+        val tmpwskprops = File.createTempFile("wskprops", ".tmp")
+        val env = Map("WSK_CONFIG_FILE" -> tmpwskprops.getAbsolutePath())
```

Comment:
We could factor this into a new `withConfigFile` pattern to reduce the number of times we end up repeating create file, set env map, and delete file on error. This reduces the changes of forgetting a finally also.